embabel-agent-api/src/main/kotlin/com/embabel/agent/spi/support/springai/streaming/StreamingChatClientOperations.kt

```diff
@@ -344,21 +344,25 @@ internal class StreamingChatClientOperations(
      */
     fun rawChunksToLines(raw: Flux<String>): Flux<String> {
         val buffer = StringBuffer()
```

Comment:
I missed this the first time around, but can't we use a non-synchronized StringBuilder here?